examples/MQ/serialization/data/MyHit.cxx

```diff
@@ -1,40 +0,0 @@
-/********************************************************************************
- *    Copyright (C) 2014 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    *
- *                                                                              *
- *              This software is distributed under the terms of the             *
- *              GNU Lesser General Public Licence (LGPL) version 3,             *
- *                  copied verbatim in the file "LICENSE"                       *
- ********************************************************************************/
-
-/*
- * File:   MyHit.cxx
- * Author: winckler
- *
- * Created on November 24, 2014, 1:40 PM
- */
-
-#include "MyHit.h"
-
-MyHit::MyHit()
-    : FairHit()
-{}
-
-MyHit::MyHit(Int_t detID, Int_t mcindex, const TVector3& pos, const TVector3& dpos)
-    : FairHit(detID, pos, dpos, mcindex)
-{}
-
-MyHit::MyHit(Int_t detID,
-             Int_t mcindex,
-             const TVector3& pos,
-             const TVector3& dpos,
-             Double_t timeStamp,
-             Double_t timeStampError)
-    : FairHit(detID, pos, dpos, mcindex)   //
-{
-    SetTimeStamp(timeStamp);
-    SetTimeStampError(timeStampError);
-}
-
-MyHit::~MyHit() {}
-
-ClassImp(MyHit);
```

Comment:
Interesting to see that the `ClassImp` isn't needed any longer… or I don't see it in the `.h` now?

I probably should read up on when we need ClassImp and when not…

Reply:
Both `ClassImp` and `ClassDef` should not be needed as far as I understand.

Reply:
@ChristianTackeGSI I don't understand it either. I consulted Dennis and Radek and both told me it should not be needed. I also tested thoroughly after the change, including the I/O that is done with this class in the example.

Reply:
Looks like `ClassImp` got lately deprecated in ROOT:
* [Doxygen docs for ClassImp in master](https://root.cern/doc/master/Rtypes_8h.html#a3f9bf160bbbb44209e95556960836b96)
* [Commit adding the deprecation doxygen label](https://github.com/root-project/root/commit/52f133b62a04ef2b05eacf4e88c5d6730688e336#diff-7fb16dd1fa5a219aef72fcc8a7536b215e7c0df6d403f612fbd97adf0c7d4db8R374)